### PR TITLE
Mobile

### PR DESCRIPTION
**Pull request by Sankalan1987**

### CODE REVIEW

File: src/app/pages/orders.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
 	commands: [
 		{
 			checkIfNOrdersPlaced: function (n) {
-				this.assert.urlEquals(this.url());
+			//	browser.assert.urlEquals(browser.url());
```
Comment (samirans89) [inline src/app/pages/orders.js:11]: remove commented lines of code.

**Comment by samirans89** [comment]
We need a handshake with TU team to commit on main as the Web automation TU courses use the same repo.

There also needs to be uniformity between the app, tests and other artefacts which are used in existing TU repos:

- browserstack-examples-appium-junit5
- browserstack-examples-appium-webdriverio
- browserstack-examples-appium-testng

File: README.md [hunk 2]
```diff
@@ -404,6 +404,45 @@ In this section, we will run the test cases to test the internally hosted websit
```
Comment (samirans89) [inline README.md:404]: remove .DS_Store

File: README.md [hunk 3]
```diff
@@ -404,6 +404,45 @@ In this section, we will run the test cases to test the internally hosted websit
...
+#### Prerequisites
+
+You need to upload the APK or IPA to BrowserStack, before you can run the test on BrowserStack. BrowserStack will provide you with an app_url which you need to use.
+
+You can upload the APK or IPA using a file on your filesystem or using a public url.
+
+cURL command:
+
+```
+curl -u "browserstack_username:browserstack_access_key" \
```
Comment (samirans89) [inline README.md:418]: which app?

for main branch, the README doc approach has to be consistent with this one below:
https://github.com/browserstack/browserstack-examples-appium-testng#prerequisites

**Review by samirans89 — CHANGES_REQUESTED** [review]

File: src/app/pages/orders.js (hunk 1 above)
Comment (Sankalan1987) [reply src/app/pages/orders.js]: Done.

File: README.md (hunk 2 above)
Comment (Sankalan1987) [reply README.md:404]: Done.

File: README.md (hunk 3 above)
Comment (Sankalan1987) [reply README.md:420]: Done

**Pull request closed without merging**